packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -119,17 +119,17 @@ curl -v -X POST \
 
 <ApiOperation method="get" url="/api/v1/inlineHooks/${id}" />
 
-##### Request Parameters
+##### Request parameters
 
 | Parameter | Description               | Param Type   | DataType   | Required |
 | --------- | ------------------------- | ------------ | ---------- | -------- |
 | `id`      | A valid Inline Hook ID.   | Path         | String     | TRUE     |
 
-##### Response Parameters
+##### Response parameters
 
 The response is an [Inline Hook object](#inline-hook-object) representing the registered Inline Hook that matches the `id` you specified.
```

Comment:
The response is an [Inline Hook object](#inline-hook-object) that represents the registered Inline Hook that matches the `id` you specify.